src/domains/wallet-app/api/webview-payment-wallet/v1/_base_policy.xml.tpl

```diff
@@ -18,18 +18,26 @@
       <set-variable name="walletId" value="@{
           return context.Request.MatchedParameters.GetValueOrDefault("walletId","");
       }" />
-      <!--
         <validate-jwt header-name="Authorization" failed-validation-httpcode="401" failed-validation-error-message="Unauthorized" require-expiration-time="true" require-scheme="Bearer" require-signed-tokens="true" output-token-variable-name="jwtToken">
           <issuer-signing-keys>
-              <key>{{wallet-jwt-signing-key}}</key>
+              <key>@(Convert.ToBase64String(Encoding.UTF8.GetBytes({{wallet-jwt-signing-key}})))</key>
```

Comment:
maybe better create signature decoding from base64 to byte array:

`var signature = new HMACSHA512(Convert.FromBase64String("{{ecommerce-io-jwt-signing-key}}")).ComputeHash(Encoding.UTF8.GetBytes($"{jwtHeaderBase64UrlEncoded}.{jwtPayloadBase64UrlEncoded}"));`

Like in this PR https://github.com/pagopa/pagopa-infra/pull/1539